src/Microsoft.Health.Fhir.Core/Features/Operations/FhirOperationDataStoreBase.cs

```diff
@@ -0,0 +1,254 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using EnsureThat;
+using Microsoft.Extensions.Logging;
+using Microsoft.Health.Fhir.Core.Features.Conformance.Serialization;
+using Microsoft.Health.Fhir.Core.Features.Operations.Export.Models;
+using Microsoft.Health.Fhir.Core.Features.Operations.Reindex.Models;
+using Microsoft.Health.Fhir.Core.Features.Persistence;
+using Microsoft.Health.JobManagement;
+using Newtonsoft.Json;
+
+namespace Microsoft.Health.Fhir.Core.Features.Operations;
+
+public abstract class FhirOperationDataStoreBase : IFhirOperationDataStore
+{
+    private readonly IQueueClient _queueClient;
+    private readonly JsonSerializerSettings _jsonSerializerSettings;
+    private readonly ILogger<FhirOperationDataStoreBase> _logger;
+
+    protected FhirOperationDataStoreBase(IQueueClient queueClient, ILoggerFactory loggerFactory)
+    {
+        _queueClient = EnsureArg.IsNotNull(queueClient, nameof(queueClient));
+        _logger = EnsureArg.IsNotNull(loggerFactory, nameof(loggerFactory)).CreateLogger<FhirOperationDataStoreBase>();
+
+        _jsonSerializerSettings = new JsonSerializerSettings
+        {
+            Converters = new List<JsonConverter>
+            {
+                new EnumLiteralJsonConverter(),
+            },
+        };
+    }
+
+    public async Task<ExportJobOutcome> CreateExportJobAsync(ExportJobRecord jobRecord, CancellationToken cancellationToken)
+    {
+        var clone = jobRecord.Clone();
+        clone.QueuedTime = DateTime.Parse("1900-01-01");
```

Comment:
Is there a reason we don't use something like DateTime.UtcNow? for the QueuedTime?

Reply:
This field is left over from the old way we handled export records. The standardized job records have queued time in them, so the specific export job record doesn't need it anymore. But we can't remove it yet because it would break backwards compatibility. So we set it to a value that is obviously wrong so that there isn't confusion on which queued time field is correct.